app/templates/static/cookies-privacy.html

```diff
@@ -78,7 +78,7 @@ <h2 class="question__title neptune">
                                         <li>Measure how you use the website so it can be updated and improved based
                                             on your needs
                                         </li>
-                                        <li>Remember the notifications you’ve seen so that we don’t show them to you
+                                        <li>Remember the notifications you've seen so that we don't show them to you
```

Comment:
@armstrongb @boxadesign I thought these should have been the smart quotes `’` instead of the straight ones `'`. Thats the standard we have across survey runner

Reply:
OK I'll update

Reply:
Yes all quotes should be smart quotes. (sorry if previous comments were unclear)